docs/GettingStarted/configuration/ExporterWebhook.md

```diff
@@ -0,0 +1,203 @@
+
+While the webhook exporter can be used on its own, it can also be used in conjunction with other exporters to gather complementary data.
+
+It's important to note that the webhook exporter performs validation of the payload data to check if it complies with the proper format. However, it does not check for data integrity, which is the responsibility of the service sending the HTTP POST request. This means that values like image digests, deployment namespaces, application names, or commit hashes can be anything, and if improper values are sent, they will still be collected and may cause dashboard inconsistencies.
+
+## Example
+
+The webhook exporter configuration option must be placed under `spec.exporters.instances` in the Pelorus configuration object YAML file as in the example, with a non-default [LOG_LEVEL](#log_level) option:
+
+```yaml
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+kind: Pelorus
+metadata:
+  name: example-configuration
+spec:
+  exporters:
+    instances:
+      - app_name: webhook-exporter
+        exporter_type: webhook
+        extraEnv:
+          - name: LOG_LEVEL
+            value: debug
+```
+
+## Webhook endpoint URI
+
+When you deploy the webhook exporter, an OpenShift route with the HTTP endpoint is created. This endpoint allows services to send HTTP POST requests. To access the webhook endpoint, simply add the `/webhook/pelorus` suffix to the HTTP endpoint created during deployment.
+
+To find the URI of this route after deploying the webhook exporter, use the following `oc` command:
+
+```shell
+$ oc get routes -n pelorus webhook-exporter
+NAME               HOST/PORT             PATH   SERVICES           PORT   TERMINATION   WILDCARD
+webhook-exporter   webhook.endpoint.uri         webhook-exporter   http                 None
+```
+
+The POST webhook endpoint for the above example: `webhook.endpoint.uri/webhook/pelorus`
+
+## Configuration options
+
+This is the list of options that can be applied to `env_from_secrets`, `env_from_configmaps` and `extraEnv` section of a webhook exporter.
+
+| Variable | Required | Default Value |
+|----------|----------|---------------|
+| [LOG_LEVEL](#log_level) | no | `INFO` |
+
+###### LOG_LEVEL
+
+- **Required:** no
+    - **Default Value:** INFO
+- **Type:** string
+
+: Set the log level. One of `DEBUG`, `INFO`, `WARNING`, `ERROR`.
+
+## Webhook headers and payloads
+
+When sending an HTTP POST request to the webhook's configured URL endpoint, the payload must conform to the webhook payload specification and include several special headers. It's important to note that the header specifications may vary depending on the Pelorus plugin determined by the `User-Agent` Header value and are described per plugin, alongside the payload specification.
+
+### `Pelorus-Webhook/`
+#### Headers specification
+
+| Header | Description |
+|--------|---------------|
+| `User-Agent` | Prefix must begin with `Pelorus-Webhook/` in order to use this plugin. |
```

Comment:
did not understand this. I can put anything after and it works?

Reply:
Yes, the spec normally have just prefix, so it's same to have user-agent:

`Pelorus-Webhook/something`
`Pelorus-Webhook/something2`

Similarly it's for other services such as GitHub, please check last sentence in the paragraph:
https://docs.github.com/webhooks-and-events/webhooks/webhook-events-and-payloads#delivery-headers

Reply:
hmm it works like and id then. ok